MAUI/ComboBox/AutoSizing.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: AutoSizing in .NET MAUI ComboBox control | Syncfusion
+description: Learn all about autosizing support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more here.
```

Comment:
description: Learn about autosizing support in Syncfusion .NET MAUI ComboBox (SfComboBox) control and more.